ApigeeToAzureApimMigrationTool.Logic/ApigeeToAzureApimMigrationTool.Service.csproj

```diff
@@ -9,6 +9,7 @@
   <ItemGroup>
     <PackageReference Include="Azure.Identity" Version="1.10.2" />
     <PackageReference Include="Azure.ResourceManager.ApiManagement" Version="1.1.0" />
+    <PackageReference Include="IPNetwork" Version="1.3.2" />
```

Comment:
IPNetwork was last published in 2015.  IPNetwork2 appears to be much, much more recent and actively developed.